code42/py42
# make create just call get instead

### Description of Change ###

Instead of returning a noop 200 response, we could return the response from `get()`. That way, it will contain the same response data as before.

One issue with this is that getting a detection list profile by username is more delayed than getting a profile by ID, for newly created users. However, anybody's script that was calling `create` for newly created users would have already been failing from the API, so this is fine imo.

### Issues Resolved ###
n/a


### Testing Procedure ###
Call the method `sdk.detectionlists.create_user()` and make sure the response is 200 for users that exist.

### PR Checklist ###
Did you remember to do the below?
